themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -156,7 +156,7 @@ JS;
       <a class="sr-only" href="#content"><?=$this->transEsc('Skip to content') ?></a>
       <?=$this->render('header.phtml')?>
     </header>
-    <nav class="breadcrumbs">
+    <nav class="breadcrumbs" aria-label="<?=$this->transEsc('Breadcrumbs') ?>">
```

Comment:
Breadcrumbs needs to be added to the language file. We should probably see if there are any other new strings we've missed, too.

Reply:
I've caught one: 
`<span class="sr-only">Toggle navigation</span> `
in header.phtml is provided in English only but should be made translatable.

Reply:
Good catch, @ckaz! @crhallberg, can you add a fix for that to this PR when you correct the other outstanding translation issues?